config/database.yml

```diff
@@ -17,8 +17,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
-  username: postgres
-  password: postgres
+  username: dev
+  password: PASs123
```

Comment:
Let's in the future try to use the same username and password or maybe use env variables.